core/App/components/misc/CredentialCard11.tsx

```diff
@@ -113,33 +136,63 @@ const CredentialCard11: React.FC<CredentialCard11Props> = ({ credential, style =
       justifyContent: 'center',
       alignItems: 'center',
     },
+    headerText: {
+      ...TextTheme.labelSubtitle,
+      ...ListItems.recordAttributeText,
+      fontSize: 15,
+      flexShrink: 1,
+    },
+    valueText: {
+      ...TextTheme.normal,
+      minHeight: ListItems.recordAttributeText.fontSize,
+      paddingVertical: 4,
+    },
     textContainer: {
-      color: credentialTextColor(ColorPallet, overlay.cardLayoutOverlay?.primaryBackgroundColor),
+      color: proofItems
+        ? TextTheme.normal.color
+        : credentialTextColor(ColorPallet, overlay.cardLayoutOverlay?.primaryBackgroundColor),
       flexShrink: 1,
     },
+    errorText: {
+      ...TextTheme.normal,
+      color: ListItems.proofError.color,
+    },
+    errorIcon: {
+      color: ListItems.proofError.color,
+    },
   })
 
   useEffect(() => {
-    if (!(credential && isValidIndyCredential(credential))) {
-      return
-    }
-
     const resolveBundle = async () => {
-      const bundle = await OCABundleResolver.resolve(credential, i18n.language)
-      const defaultBundle = await OCABundleResolver.resolveDefaultBundle(credential, i18n.language)
+      let bundle: OCABundle | undefined
```

Comment:
I think we still need the check when the credential is defined, so I would turn this block into:

```ts
let bundle = await OCABundleResolver.resolveByCredDefOrSchema(credDefId, schemaId, i18n.language)
let defaultBundle = await OCABundleResolver.resolveDefaultBundleByCredDefOrSchema(credDefId, schemaId, credName, i18n.language)

if (credential && isValidIndyCredential(credential)) {
  bundle = await OCABundleResolver.resolve(credential, i18n.language)
  defaultBundle = await OCABundleResolver.resolveDefaultBundle(credential, i18n.language)
}

```